### DISCUSSION TITLE
Welcome to tuxguitar Discussions!

### DISCUSSION THREAD

**Discussion by helge17** [helge17/tuxguitar#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community.

**Comment by BhanteWayne** [comment]
I just installed tux 1.6.6 from a .deb file.
Everything works except the metronome option.
The metronome option can be selected, under the Player tab, but there are no clicks produced.
Worked fine under 1.5.5 but it's  not functioning under 1.6.6

**Reply by guiv42** [reply]
Can you please precise:
- which operating system? (name and version)
- is it systematic? (did you test this on several tg files?)

**Reply by BhanteWayne** [reply]
The issue was slow synapses on my part.  There is no bug or problem.
I finally remembered that the metronome only works when you display an instrument.  Added a fretboard and the metronome works.
My OS is Ubuntu 24.04.2.  I tested a few 1.6 releases, chasing a bug that doesn't exist.
Thanks for your quick followup.

**Comment by TheEllice** [comment]
Hello! i recently upgraded all the way from 1.5.6 to 1.6.6 and im so far loving the new features! however im really missing the "transcibe notes" check box when changing tuning, i used it quite often in janky way to transcribe some note up or down a few steps. is this button gone forever?

**Reply by guiv42** [reply]
> is this button gone forever?

Yes, I have no intention to put these old options back in the dialog. It's really a mess to try to handle simultaneously tuning change and transposition (there were undesirable side effects).
But there should be other ways to reach your  goal, in 2 steps: see menu "Tools -> Transpose". Several use cases are also described in [this discussion](https://github.com/helge17/tuxguitar/issues/644#issuecomment-2613865511).
If it does not answer your question, then please provide a detailed description of what you want to do exactly.
